sherlock-webhook-proxy/pkg/handler.go

```diff
@@ -255,9 +256,15 @@ func HandleWebhook(w http.ResponseWriter, r *http.Request) {
 			})
 
 			// Handle response cases
+			payloadPretty, err2 := json.MarshalIndent(payload, "", "    ")
+			payloadPrettyString := ""
```

Comment:
```suggestion
			var payloadPrettyString string
```
**nitpick (non-blocking):** no need to assign to empty (I think these will compile identically but it's just a style thing)